dev/test/query.ts

```diff
@@ -658,6 +658,38 @@ describe('query interface', () => {
       expect(posts.docs[0].data().toString()).to.equal('post, by author');
     });
   });
+
+  it('for withConverter() propagation through queryOptions', async () => {
```

Comment:
The test name could be improved. The  concatenated name is now "query interface for withConverter() propagation through queryOptions".

What do you think about renaming the test to something like "propagates withConverter() through QueryOptions"?